Translations/make_translation.py

```diff
@@ -242,14 +247,24 @@ def getFontMapAndTable(textList):
     totalSymbolCount = len(set(textList) | set(forcedFirstSymbols))
     # \x00 is for NULL termination and \x01 is for newline, so the maximum
     # number of symbols allowed with 8 bits is `256 - 2`.
-    if totalSymbolCount > (256 - 2):
+    if totalSymbolCount > ((0xEE * 0x0F) - 2):

```

Comment:
Wow, from 255 to 3569, impressive

Reply:
I mean, I don't think we can get there but also would rather not revisit at 500 🤣